openassessment/xblock/test/test_grade.py

```diff
@@ -322,26 +322,35 @@ def test_grade_waiting(self, xblock, data):
 
     @scenario('data/grade_incomplete_scenario.xml', user_id='Bunk')
     def test_grade_incomplete_missing_self(self, xblock):
-        # Graded peers, but haven't completed self assessment
-        self.create_submission_and_assessments(
-            xblock, self.SUBMISSION, [self.PEERS[0]], [PEER_ASSESSMENTS[0]], None
-        )
-        resp = self.request(xblock, 'render_grade', json.dumps(dict()))
+        resp = self._test_incomplete_helper(xblock, [self.PEERS[0]], None)
+        self.assertNotIn('peer assessment', resp)
```

Comment:
If I'm reading this correctly, it is verifying that the learner is not told to do a peer assessment. However, I don't see anything verifying that they are told to complete their self assessment.